### PR TITLE
Changed header

### PR DESCRIPTION
**Pull request by inkerton**
**BEFORE**

image

<hr/>
**AFTER**

https://github.com/user-attachments/assets/a1aba46f-471f-461a-89da-c461e7a8de2d

### CODE REVIEW

**Comment by inkerton** [comment]
@anuj123upadhyay This is in response to the issue #159 

Under the blogs where there are components listed I will add blog categories there later first category needs to be added in add post and saved in database.

**Comment by anuj123upadhyay** [comment]
Hey @inkerton I will merge after 2pm

**Comment by inkerton** [comment]
@anuj123upadhyay No issues take your time let me know when you have merged it I saw 2-3 blog websites I have some more Ideas I will work after you merge these PRs. Thankx a lot for your work :)

**Review by anuj123upadhyay — APPROVED** [review]

**Pull request merged**
